src/Blueprint.h

```diff
@@ -25,465 +25,484 @@
-     *
-     *  FIXME: Use C++11 template aliases when migrating to C++11.
-     */
+/**
+ * Default Container for collections.
+ *
+ *  FIXME: Use C++11 template aliases when migrating to C++11.
+ */
 
-    template<typename T>
-    struct Collection {
-        typedef std::vector<T> type;
-        typedef typename std::vector<T>::iterator iterator;
-        typedef typename std::vector<T>::const_iterator const_iterator;
-    };
+template <typename T> struct Collection
+{
+    typedef std::vector<T> type;
+    typedef typename std::vector<T>::iterator iterator;
+    typedef typename std::vector<T>::const_iterator const_iterator;
+};
 
-    /** Metadata Collection */
-    typedef Collection<Metadata>::type MetadataCollection;
+/** Metadata Collection */
+typedef Collection<Metadata>::type MetadataCollection;
 
-    /** Headers */
-    typedef Collection<Header>::type Headers;
+/** Headers */
+typedef Collection<Header>::type Headers;
 
-    /** Collection of Parameter Values */
-    typedef Collection<Value>::type Values;
+/** Collection of Parameter Values */
+typedef Collection<Value>::type Values;
 
-    /** Role of an element */
-    enum AssetRole {
-        UndefinedAssetRole = 0,   // Unknown
-        BodyExampleAssetRole,     // Body example for asset
-        BodySchemaAssetRole       // Body schema for asset
-    };
+/** Role of an element */
+enum AssetRole
+{
+    UndefinedAssetRole = 0, // Unknown
+    BodyExampleAssetRole,   // Body example for asset
+    BodySchemaAssetRole     // Body schema for asset
+};
 
-    /** Parameter Use flag */
-    enum ParameterUse {
-        UndefinedParameterUse,
-        OptionalParameterUse,
-        RequiredParameterUse
-    };
+/** Parameter Use flag */
+enum ParameterUse
+{
+    UndefinedParameterUse,
+    OptionalParameterUse,
+    RequiredParameterUse
+};
 
-    /** Parameter */
-    struct Parameter {
+/** Parameter */
+struct Parameter
+{
 
-        /** Parameter Name */
-        Name name;
+    /** Parameter Name */
+    Name name;
 
-        /** Parameter Description */
-        Description description;
+    /** Parameter Description */
+    Description description;
 
-        /** Type */
-        Type type;
+    /** Type */
+    Type type;
 
-        /** Required flag */
-        ParameterUse use;
+    /** Required flag */
+    ParameterUse use;
 
-        /** Default Value, applicable only when `required == false` */
-        Value defaultValue;
+    /** Default Value, applicable only when `required == false` */
+    Value defaultValue;
 
-        /** Example Value */
-        Value exampleValue;
+    /** Example Value */
+    Value exampleValue;
 
-        /** Enumeration of possible values */
-        Values values;
-    };
+    /** Enumeration of possible values */
+    Values values;
+};
 
-    /** Parameter with MSON-like syntax */
-    struct MSONParameter : public Parameter {
-    };
+/** Parameter with MSON-like syntax */
+struct MSONParameter : public Parameter
+{};
```

Comment:
This is strange. Compare with
https://github.com/apiaryio/snowcrash/pull/434/files#diff-2c45c82fe0dd50c00a34ebdfb3dec1f9R70

```
SourceMap<Element>::~SourceMap() {}
```

Why are in this case empty brackets on separated line, while second case it is on same line